packages/muze/src/canvas/component-wrapper-maker.js

```diff
@@ -108,70 +175,73 @@ const scrollBarMap = config => ({
  * @return {Instance} Returns the respective wrappers
  */
 const createScrollBarWrapper = (scrollBarType, layoutManager, renderDetails, grid) => {
-    let scrollBarWrapper = null;
-    const { layoutConfig } = renderDetails;
-    const target = { target: CANVAS };
-    const { scrollBar } = layoutConfig;
-    const { layoutDimensions } = grid.viewInfo();
-    const {
-        actualCenterMeasures,
-        unitHeights,
-        unitWidths
-    } = layoutDimensions;
-    const {
-        componentName, layoutBasedMeasure, width, height, rowAlign, colAlign,
-        viewMeasure, position, alignment
-    } = scrollBarMap(scrollBar[scrollBarType])[scrollBarType];
-    const dimensions = {
-        thickness: scrollBar.thickness,
-        layoutBasedMeasure: layoutDimensions[viewMeasure][1]
-    };
-    const isScroll = grid.scrollInfo()[scrollBarType];
-    const totalLength = actualCenterMeasures[layoutBasedMeasure];
-    const viewLength = layoutDimensions[viewMeasure][1];
+    if (isGridPresent(renderDetails)) {
```

Comment:
The `sGridPresent` function is being called for multiple wrappers. 
It's better to call the function once and then pass the result to the wrappers which can then use the result instead of the function being executed multiple times.